docs/reference/README.md

```diff
@@ -0,0 +1,21 @@
+# API Reference
+
+This is the API Reference for Piraeus Operator. It lists user settable options that control the state of
+Piraeus Datastore.
+
+### [`LinstorCluster`](./linstorcluster.md)
+
+This resource controls the state of the LINSTOR&reg cluster and integration with Kubernetes.
```

Comment:
`&reg` doesn't render properly in GitHub: https://github.com/piraeusdatastore/piraeus-operator/tree/2b7dd10f65197ec41d78cee1de3032c33b2b5a56/docs/reference

Reply:
Looke like the linter lied to me :sweat_smile: